Content.Shared/Resist/EscapeInventoryCancelEvent.cs

```diff
@@ -0,0 +1,6 @@
+using Content.Shared.Actions;
```

Comment:
shouldnt this entire file be done in delta V namespace aka done in delta V folder?

Reply:
I'm not sure it's worth giving just one single event that's only used in *changed upstream code* a dedicated deltav namespace. But if other people share the same opinion, I can move it.